chapter06/docker-compose.yml

```diff
@@ -123,7 +112,7 @@ services:
     <<: *airflow-common
     command: webserver
     ports:
-      - "8080:8080"
+      - "8082:8080"
```

Comment:
Why do we need a different port here. All other compose files use 8080

Reply:
indeed